docs/_data/navigation.yml

```diff
@@ -1,109 +1,38 @@
 main:
   - title: Docs
-    url: /docs/overview.html
+    url: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
+    target: target="_blank" rel="noopener noreferrer"
     section: /docs/
-    sublinks:
-    - title: Overview
-      url: /docs/overview.html
-      subsublinks:
-      - title: Why Secretless?
-        url: /docs/overview/why_secretless.html
-      - title: How Does It Work?
-        url: /docs/overview/how_it_works.html
-      - title: Key Terms
-        url: /docs/overview/key_terms.html
-      - title: Security
-        url: /docs/overview/security.html
-    - title: Get Started
-      url: /docs/get_started.html
-    - title: Reference
-      url: /docs/reference.html
-    - title: FAQ
-      url: /docs/faq.html
   - title: Community
     url: /community.html
   - title: Blog
     url: /blog.html
   - title: GitHub
     url: https://github.com/cyberark/secretless-broker
     class: gh
-    target: target="_blank"
+    target: target="_blank" rel="noopener noreferrer"
 
-chapters:
-  - title: Overview
-    url: /docs/overview.html
-    sections:
-      - page: Why Secretless?
-        url: /docs/overview/why_secretless.html
-      - page: How Does It Work?
-        url: /docs/overview/how_it_works.html
-      - page: Key Terms
-        url: /docs/overview/key_terms.html
-      - page: Security
-        url: /docs/overview/security.html
-  - title: Get Started
-    url: /docs/get_started.html
-    sections:
-      - page: Quick Start
-        url: /docs/get_started/quick_start.html
-      - page: Kubernetes Tutorial
-        url: /docs/get_started/kubernetes_tutorial.html
-      - page: Configuration
-        url: /docs/get_started/configuration.html
-      - page: How to Deploy
-        url: /docs/get_started/how-to-deploy/overview.html
-        entries:
-        - page: Using Conjur
-          url: /docs/get_started/how-to-deploy/using-conjur.html
-        - page: Using Kubernetes Secrets
-          url: /docs/get_started/how-to-deploy/using-kubernetes-secrets.html
+footer-docs:
+  - title: Why Secretless?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
```

Comment:
will remove staging from here when docs site is ready

Reply:
@sigalsax I think it's ok to remove it now - afaik I'm the only one who'd try to click on any links from this PR, and I can manually add staging if needed to access a valid URL